index.bs

```diff
@@ -212,21 +212,27 @@ The <dfn>AV1CodecConfigurationBox</dfn> contains decoder configuration informati
 ### Syntax ### {#av1codecconfigurationbox-syntax}
 
 ```
-class AV1CodecConfigurationBox extends FullBox('av1C', version = 0, 0){
+class AV1CodecConfigurationBox extends FullBox('av1C', version = 1, 0){
   AV1CodecConfigurationRecord av1Config;
 }
```

Comment:
25 bits ?
The struct OBU array is no longer aligned to 8 bits.

Reply:
Thanks-- Didn't account for one of the flags (initial_presentation_delay_present or sequence_parameters_present) when I adjusted the size of the first reserved field.